aws-cpp-sdk-core/CMakeLists.txt

```diff
@@ -402,6 +397,13 @@ else()
     endif()
 endif()
 
+configure_file("${CMAKE_SOURCE_DIR}/toolchains/pkg-config.pc.in" "${PROJECT_NAME}.pc" @ONLY)
```

Comment:
Right, so this was moved because we'd want to retain the pc functionality even on non SIMPLE_INSTALL? If the changes are to be gated on SIMPLE_INSTALL then it should lie in the appropriate if expression block. I wasn't sure what you'd want to gate on SIMPLE INSTALL so I've left it out of the if blocks for now.